WireDomain/Sources/WireDomain/Synchronization/PullAllConversationsSync.swift

```diff
@@ -19,29 +19,32 @@
 import Foundation
 import WireAPI
 
-struct PullAllConversationsSync: PullAllConversationsSyncProtocol {
+public final class PullAllConversationsSync: PullAllConversationsSyncProtocol {
```

Comment:
question: why does it become a class

Reply:
to mutate the journal. Maybe the journal can be a class instead?